lib/storage.ex

```diff
@@ -1,12 +1,11 @@
 defmodule Pepe.Storage do
```

Comment:
I didn't moved this to `lib/storage/` to easy review (otherwise git would show a deletion and a new file with all the code)

But I intend to do it after approval